index.js

```diff
@@ -119,6 +123,10 @@ if (dryRun) {
   console.log('Dry-run enabled!');
 }
 
+var cutoffDateMillis = new Date().getTime() - numDaysOld * NUM_MILLIS_IN_DAY;
+var cutoffDate = moment(new Date(cutoffDateMillis)).format('YYYY-MM-DD');
```

Comment:
Also, I believe you can do this with just moment, without `NUM_MILLIS_IN_DAY`, with just `moment().subtract(numDaysOld, 'days')`